addons/account/models/account_move.py

```diff
@@ -541,6 +541,12 @@ def _sequence_fixed_regex(self):
         store=True,
         readonly=False,
     )
+
+    reconciliation_model_id = fields.Many2one(
+        string='Reconciliation model',
+        comodel_name='account.reconcile.model',
```

Comment:
Since the lines are already linked to the reco model... I'm not sure we really need this field